x/evm/keeper/integration_test.go

```diff
@@ -69,16 +69,6 @@ var _ = Describe("Handling a MsgEthereumTx message", Label("EVM"), Ordered, func
 	})
 
 	When("the params have default values", Ordered, func() {
-		BeforeAll(func() {
```

Comment:
Hmm this might be unrelevant but they assure that independently of the order of the tests, this `When` block is always tested against default params

Reply:
added it back